deployments/bridges/rococo-wococo/dashboard/grafana/bridges-alerts.json

```diff
@@ -1,1994 +1 @@
01) node is down"},"grafana_alert":{"id":74,"orgId":1,"title":"RococoBridgeHub \u003c\u003e WococoBridgeHub relay (00000001) node is down","condition":"C","data":[{"refId":"A","queryType":"","relativeTimeRange":{"from":900,"to":0},"datasourceUid":"PC96415006F908B67","model":{"datasource":{"type":"prometheus","uid":"PC96415006F908B67"},"editorMode":"code","exemplar":false,"expr":"up{domain=\"parity-testnet\",container=\"bridges-common-relay\"}","instant":false,"interval":"","intervalMs":30000,"legendFormat":"Is relay running","maxDataPoints":43200,"range":true,"refId":"A"}},{"refId":"B","queryType":"","relativeTimeRange":{"from":0,"to":0},"datasourceUid":"-100","model":{"conditions":[{"evaluator":{"params":[],"type":"gt"},"operator":{"type":"and"},"query":{"params":["B"]},"reducer":{"params":[],"type":"last"},"type":"query"}],"datasource":{"type":"__expr__","uid":"-100"},"expression":"A","hide":false,"intervalMs":1000,"maxDataPoints":43200,"reducer":"max","refId":"B","type":"reduce"}},{"refId":"C","queryType":"","relativeTimeRange":{"from":0,"to":0},"datasourceUid":"-100","model":{"conditions":[{"evaluator":{"params":[1],"type":"lt"},"operator":{"type":"and"},"query":{"params":["C"]},"reducer":{"params":[],"type":"last"},"type":"query"}],"datasource":{"type":"__expr__","uid":"-100"},"expression":"B","hide":false,"intervalMs":1000,"maxDataPoints":43200,"refId":"C","type":"threshold"}}],"updated":"2023-04-18T09:14:37Z","intervalSeconds":60,"version":4,"uid":"9YAdEUB4z","namespace_uid":"eblDiw17z","namespace_id":140,"rule_group":"Bridges","no_data_state":"OK","exec_err_state":"OK"}}]},{"name":"Rococo Beefy best blocks not advancing","interval":"20m","rules":[{"expr":"","for":"1h","labels":{"matrix_room":"!lMunCqbBqxEqfRuUDF:parity.io"},"annotations":{"__dashboardUid__":"3sEDRyl7z","__panelId__":"2","summary":"Rococo BEEFY best blocks have not advanced for at least 60 mins"},"grafana_alert":{"id":41,"orgId":1,"title":"Rococo BEEFY best blocks not advancing","condition":"C","data":[{"refId":"A","queryType":"","relativeTimeRange":{"from":10800,"to":0},"datasourceUid":"PC96415006F908B67","model":{"editorMode":"code","expr":"increase(substrate_beefy_best_block{chain=\"rococo_v2_2\", node=~\"rococo.*(3-validator|3-rpc).*\"}[1h])","hide":false,"intervalMs":1000,"maxDataPoints":43200,"range":true,"refId":"A"}},{"refId":"C","queryType":"","relativeTimeRange":{"from":0,"to":0},"datasourceUid":"-100","model":{"conditions":[{"evaluator":{"params":[100],"type":"lt"},"operator":{"type":"and"},"query":{"params":["A"]},"reducer":{"params":[],"type":"last"},"type":"query"}],"datasource":{"type":"grafana-expression","uid":"-100"},"hide":false,"intervalMs":1000,"maxDataPoints":43200,"refId":"C","type":"classic_conditions"}}],"updated":"2023-02-14T12:26:10Z","intervalSeconds":1200,"version":10,"uid":"CBuugs_7k","namespace_uid":"eblDiw17z","namespace_id":140,"rule_group":"Rococo Beefy best blocks not advancing","no_data_state":"Alerting","exec_err_state":"Alerting"}}]}]}
```

Comment:
@svyatonik 
json unprettying was intentional?

Reply:
Thanks. Probably I prettified it manually last time. Grafana returns unprettified json.